python/cudf_polars/tests/test_scan.py

```diff
@@ -332,12 +332,17 @@ def test_scan_parquet_only_row_index_raises(df, tmp_path):
     assert_ir_translation_raises(q, NotImplementedError)
 
 
-def test_scan_include_file_path(tmp_path, format, scan_fn, df):
+def test_scan_include_file_path(request, tmp_path, format, scan_fn, df):
     make_source(df, tmp_path / "file", format)
 
     q = scan_fn(tmp_path / "file", include_file_paths="files")
 
-    assert_ir_translation_raises(q, NotImplementedError)
+    if format == "ndjson":
+        assert_ir_translation_raises(q, NotImplementedError)
+    elif format == "parquet":
+        assert_gpu_result_equal(q, engine=NO_CHUNK_ENGINE)
```

Comment:
*question*: I think this means that the chunked parquet reader doesn't support `include_file_paths`. But I didn't see that raising in translation?

Reply:
`chunked_parquet` hits the fallthrough.